docs/MongodbEventRepository.md

```diff
@@ -54,13 +54,23 @@ class PersonRepository extends MongodbEventRepository<Person> {
 
     return { entities, count, range, total }
   }
+
+  async existBy(fields){
```

Comment:
Can you add the method on the method list with description on the top of the file?

Reply:
done